index.html

```diff
@@ -204,50 +216,53 @@ <h2 class="project_title">
-              </p>
-              <h2 class="box3">
-                  LET'S CONNECT
-              </h2>
-              <ul class="box4">
-                   <li class="socials">
-                      <a href="https://twitter/harlexkhal">
-                        <img src="images/twitter.png" alt="twitter">
-                       </a>
-                   </li>
-                   <li class="socials">
-                     <a href="https://www.linkedin.com/in/alexander-oguzie-ibeh-776814164">
-                       <img src="images/Linkedin.png" alt="Linkedin">
-                     </a>
-                   </li>
-                   <li class="socials child">
-                     <a href="#">
-                       <img src="images/m.png" alt="M">
-                     </a>
-                   </li>
-                   <li class="socials">
-                     <a href="https://github.com/harlekhal">
-                       <img src="images/github.png" alt="github">
-                     </a>
-                   </li>
-                   <li class="socials">
-                     <a href="#">
-                       <img src="images/peace.png" alt="peace">
-                     </a>
-                   </li>
-              </ul>
-               <button type="button" class="resume_btn">
-                  Get my resume
-               </button>
+               </h1>
+                <p class="box2">
+                  Hello I'm a software developer! I can help you build a product, 
+                  feature or website Look through some of my work and experience! 
+                  If you like what you see and have a project you need coded, don't hestiate to contact me.
+                </p>
+                <h2 class="box3">
+                    LET'S CONNECT
+                </h2>
+                <ul class="box4">
+                    <li class="socials">
+                        <a href="https://twitter/harlexkhal">
+                          <img src="images/twitter.png" alt="twitter">
+                        </a>
+                    </li>
+                    <li class="socials">
+                      <a href="https://www.linkedin.com/in/alexander-oguzie-ibeh-776814164">
+                        <img src="images/Linkedin.png" alt="Linkedin">
+                      </a>
+                    </li>
+                    <li class="socials">
+                      <a href="#">
+                        <img src="images/m.png" alt="M">
+                      </a>
+                    </li>
+                    <li class="socials">
+                      <a href="https://github.com/harlekhal">
+                        <img src="images/github.png" alt="github">
+                      </a>
+                    </li>
+                    <li class="socials">
+                      <a href="#">
+                        <img src="images/peace.png" alt="peace">
+                      </a>
+                    </li>
```

Comment:
- [ ] These icons ended up beautiful! 💯 Let's now please kindly align these icons to the left, matching the alignment of the other blocks of text, like in the design.

![image](https://user-images.githubusercontent.com/57421823/149131544-76dda5fe-c00f-441a-80ae-b1eec01b94ec.png)